bindinfo/handle.go

```diff
@@ -169,6 +169,79 @@ func (h *BindHandle) Update(fullLoad bool) (err error) {
 	return nil
 }
 
+// CreateBindRecord creates a BindRecord to the storage and the cache.
+// It replaces all the exists bindings for the same normalized SQL.
+func (h *BindHandle) CreateBindRecord(sctx sessionctx.Context, record *BindRecord) (err error) {
+	err = record.prepareHints(sctx)
+	if err != nil {
+		return err
+	}
+
+	exec, _ := h.sctx.Context.(sqlexec.SQLExecutor)
+	h.sctx.Lock()
+	_, err = exec.ExecuteInternal(context.TODO(), "BEGIN")
+	if err != nil {
+		h.sctx.Unlock()
+		return
+	}
+
+	br := h.GetBindRecord(parser.DigestNormalized(record.OriginalSQL), record.OriginalSQL, record.Db)
```

Comment:
```suggestion
	normalizedSQL := parser.DigestNormalized(record.OriginalSQL)
	br := h.GetBindRecord(normalizedSQL, record.OriginalSQL, record.Db)
```

Reply:
so that `normalizedSQL` can be reused in the following, no need to calculate the normalized sql repeatedly.